### ISSUE TITLE
Invoice amount is not included in QR Code

### ISSUE THREAD

**Issue by benjaminvoelkl** [btcpayserver/btcpayserver#26]
Currently scanning the QR Code just returns the adress, not the due amount in BTC.

So the user has to manualy include the requested amount, instead of getting the value prefilled in his wallet.

 This also applies to the button "open with wallet"

**Comment by NicolasDorier** [comment]
Confirmed will fix today.

**Comment by NicolasDorier** [comment]
Fixed in `1.0.0.53`. Update your docker-compose and restart the service.
If you are using it on azure, just run.

```
cd `dirname $BTCPAY_DOCKER_COMPOSE` && git pull && docker-compose -f $BTCPAY_DOCKER_COMPOSE down && docker-compose -f $BTCPAY_DOCKER_COMPOSE up -d
```

**Comment by NicolasDorier** [comment]
Reopen if issue persist in 1.0.0.53
